process-compose/suzuka-full-node/process-compose.hasura.yml

```diff
@@ -0,0 +1,12 @@
+version: "3"
+
+processes:
+
+  hasura:
+    command: |
+      EXTERNAL_IP=$POSTGRES_HOST_IP docker compose -f docker/compose/indexer/docker-compose.hasura.yml up --force-recreate --renew-anon-volumes
```

Comment:
Do we want to be able to test against already filled volumes? Should we provide an option to run without --renew-anon-volumes?

Reply:
It would good to...

1. Add a running with this to the CI.
2. Add some e2e tests. For this PR, as simple as we can make them.

Reply:
About the `--renew-anon-volumes` I wasn't sure, the idea was to be sure the environment is always the same when you start, even if you've started several times before. I'm not sure the Hasura docker script use or not anonymous volumes. If you find it unnecessary, we should remove it.

Reply:
I remove the  `--renew-anon-volumes`

Reply:
I've added the e2e test to the CI.